diff --git a/comfy/samplers.py b/comfy/samplers.py
index 17f7bb8..87c1a91 100755
--- a/comfy/samplers.py
+++ b/comfy/samplers.py
@@ -436,7 +436,7 @@ def _calc_cond_batch_multigpu(model: BaseModel, conds: list[list[dict]], x_in: t
             to_batch_temp.reverse()
             to_batch = to_batch_temp[:1]
 
-            free_memory = comfy.model_management.get_free_memory(current_device)
+            free_memory = model_options['multigpu_clones'][current_device].get_free_memory(current_device)
             for i in range(1, len(to_batch_temp) + 1):
                 batch_amount = to_batch_temp[:len(to_batch_temp)//i]
                 input_shape = [len(batch_amount) * first_shape[0]] + list(first_shape)[1:]
